overlord/snapstate/snapstate_update_test.go

```diff
@@ -15803,6 +15830,36 @@ func (s *snapmgrTestSuite) testUpdateWithComponentsRunThrough(c *C, opts updateW
 	})
 	c.Assert(err, IsNil)
 
+	// Check boundaries are in the expected tasks
```

Comment:
should we Get("restart-boundary"...) and check it directly as well?

Reply:
Done